src/styles/data-vis.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .axesntricks-images{
-  margin-top: -60px;
+  margin-top: -60px !important;
```

Comment:
We try to avoid using important wherever possible.

Reply:
If it’s not possible, a line comment with an explanation will do.

Reply:
Actually, I was testing and not using it, removing that style.